src/components/calculator.js

```diff
@@ -0,0 +1,45 @@
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator-grid">
+      <div className="output">
+        <div className="previous-operand">1234+</div>
+        <div className="current-operand">1234</div>
+      </div>
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="colored">
+        ÷
+      </button>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="colored">
+        ×
+      </button>
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="colored">
+        -
+      </button>
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="colored">
+        +
+      </button>
+      <button type="button" className="span-two">
+        0
+      </button>
+      <button type="button">.</button>
+      <button type="button" className="colored">
+        =
+      </button>
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
- [ ] It is required that the `Calculator.js` component should have at least 1 child component. Something like this. :point_down: 

```
  function Buttons(props) {
      const { digit } = props
       return <div>{digit}</div>
  }

   function Calculator() {
       <div className="container">
          <div className="screen"><p className="number">0</p> 
          </div>
          <div className="buttons">
             <Buttons digit="AC" />
             <Buttons digit="±" />
             .....etc
          </div>    
        
   }
```
Kindly look into that and fix.

![child](https://user-images.githubusercontent.com/26597432/226595393-5ec8c6d5-dd71-4e96-9c07-42a9f5f96eaf.png)